.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,5 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
should this be 3.10?

Reply:
pywatershed's `pyproject.toml` has a minimum of 3.10 in `requires-python` but the classifiers don't reflect that (or the maximum), maybe that's where this is pulled from?